### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by saifrahmn**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1191"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]

**Review by krshdev — APPROVED** [review]

File: public/data/saifrahman.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      {
+        "name": "Contact me",
+        "url": "mailto:saifrahmn78@gmail.com",
+        "icon": "Envelope"
+      },
+      {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/saif-rahman-a62686201",
+      "icon": "LinkedIn"
+    },
```
Comment (schmelto) [inline public/data/saifrahman.json:37]:
```suggestion
      },
```

**Comment by schmelto** [comment]
Further there is a merge conflict could you try to resolve it. If you need any help please let us know.

**Comment by schmelto** [comment]
Merge conflict still occurs 🙈

**Review by schmelto — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Panquesito7 — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]

**Review by emmalearnscode — APPROVED** [review]

**Pull request merged**